profiles/views.py

```diff
@@ -84,12 +86,27 @@ class ProfileDetail(RetrieveUpdateDestroyAPIView):
     serializer_class = ProfileSerializer
     permission_classes = (IsAuthenticated,)
 
-    def get_queryset(self):
+    def get_queryset(self, pk=None):
         user_id = self.request.user.id
         if self.request.method == "DELETE":
             return Profile.objects.filter(person_id=user_id, is_deleted=False)
         if self.request.method == 'GET':
             return Profile.objects.filter(is_deleted=False)
+        if self.request.method == 'PUT':
```

Comment:
You can combine these two, since they return the same objects. I suggest `if self.request.method in ["PUT", "PATCH"]`

Reply:
You are right. Thank you. Done.